lib/validators/passwordValidator.ts

```diff
@@ -0,0 +1,9 @@
+import * as Yup from 'yup'
+
+export const passwordValidator = Yup.string()
+  .required('Passwort muss angegeben werden.')
+  .max(128, 'Das Passwort darf nicht länger als 128 Zeichen sein.')
```

Comment:
Devise in the backend has a max length for passwords

Reply:
I feel like this gh comment would be more useful as a ts comment :grin: